### PR TITLE
Acronyms and abbreviations from Hut23 wiki to Handbook

### PR DESCRIPTION
**Pull request by dsj976**
## Summary

This PR moves the section "Acronyms and Abbreviations" from the Hut23 wiki to the Handbook.

This content has been moved to a "Miscellaneous" section within the Handbook. Please suggest a more appropriate location if you can think of one.

**Note**: the "Miscellaneous" section is just a temporary name that is likely to change as more content is moved from the wiki into the Handbook.

## Related issues

Contributes to alan-turing-institute/Hut23#1621

## Screenshots

---

### Updates

Finally this page has been placed in the "How we work" section of the Handbook.

### CODE REVIEW

File: content/docs/miscellaneous/acronyms_and_abbreviations.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Acronyms and Abbreviations
+
+AIDA: Artificial Intelligence for Data Analytics
+ALOGIT: name of a Fortran package (see [issue](https://github.com/alan-turing-institute/Hut23/issues/295))
+ASG: AI for Science and Government (the new name of the EPSRC funds from SPF)
```
Comment (dsj976) [inline content/docs/miscellaneous/acronyms_and_abbreviations.md:18]: ASG has concluded - should we remove it?

File: content/docs/miscellaneous/acronyms_and_abbreviations.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Acronyms and Abbreviations
+
+AIDA: Artificial Intelligence for Data Analytics
```
Comment (dsj976) [inline content/docs/miscellaneous/acronyms_and_abbreviations.md:16]: AIDA was a small project a number of years ago - should we remove it?

**Comment by dsj976** [comment]
@Iain-S I have asked your review because you were one of the people who edited this section in the Hut23 wiki (the other was Roly, hence why I have also asked for your review @triangle-man). Please let me know if you are happy with this section as is or otherwise suggest some edits. Thanks.

File: content/docs/miscellaneous/acronyms_and_abbreviations.md (hunk 1 above)
Comment (mhauru) [reply content/docs/miscellaneous/acronyms_and_abbreviations.md:18]: People might still refer to it when talking about old projects.

**Pull request merged**